_includes/header--help.html

```diff
@@ -6,7 +6,7 @@
           <img
             src="{{ site.baseurl }}/assets/img/logo.svg"
             class="usa-logo__img"
-            alt="login.gov home"
+            alt="{{ site.data.[page.lang].settings.identifier.accessible_labels.login_logo }}"
```

Comment:
Glad we're translating these now 🙌

Reply:
1st part. Understood
2nd part. Are we saying maybe we say "homepage link" instead of just Homepage? but wouldn't the screen readers announce its a link already since its wrapped in an A tag.

Reply:
Oh, sorry, my message was confusing. I was specifically referring to how we name the YAML key, not the text shown to users. Since both LG-9621 and LG-9623 will add different text alternatives for logos, we'd want some way to tell them apart, where the difference is that these images in LG-9621 are used in links. 

(In fact, my impression is that it doesn't really matter that they're images/logos in how they end up being used, since they're more relevant as the text of the _link_, not the image)

Reply:
got ya. I did that already in my previous ticket. goin to ensure things work as expected.